pkg/packageinstall/packageinstall_downgrade_test.go

```diff
@@ -188,7 +188,7 @@ func Test_PackageInstallVersionDowngrades(t *testing.T) {
 					Message: "Error (see .status.usefulErrorMessage for details)",
 				}},
 				FriendlyDescription: "Reconcile failed: Error (see .status.usefulErrorMessage for details)",
-				UsefulErrorMessage:  "Stopped installing matched version '1.0.0' since last attempted version '2.0.0' is higher",
+				UsefulErrorMessage:  "Stopped installing matched version '1.0.0' since last attempted version '2.0.0' is higher.\nAdd annotation packaging.carvel.dev/downgradable: \"\" to PackageInstall to proceed with downgrade",
```

Comment:
nit: i think we should follow `(hint: Add ...)` format here similar to other carvel tools.